superset-frontend/src/explore/components/controls/ContourControl/ContourOption.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with work for additional information
+ * regarding copyright ownership.  The ASF licenses file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { styled } from '@superset-ui/core';
+import { ContourOptionProps } from './types';
+import ContourPopoverTrigger from './ContourPopoverTrigger';
+import OptionWrapper from '../DndColumnSelectControl/OptionWrapper';
+
+const StyledOptionWrapper = styled(OptionWrapper)`
+  max-width: 100%;
+  min-width: 100%;
+`;
+
+const StyledListItem = styled.li`
+  display: flex;
+  align-items: center;
+`;
+
+const ColorPatch = styled.div<{ formattedColor: string }>`
+  background-color: ${({ formattedColor }) => formattedColor};
+  height: 5px;
```

Comment:
Could we use grid units from the theme here? Like `${({ theme }) => theme.gridUnit }px;`. 1 grid unit is 4px and we prefer using whole units, so it's going to be 4px instead of 5 but I guess that shouldn't make much difference here

Reply:
Thanks for the pointer! I wasn't aware of theme grid units!